test/e2e/common_suite_test.go

```diff
@@ -203,6 +219,15 @@ func (tc *testCase) run() {
 					}
 					t.Logf("Log output of peer pod:%s", LogString)
 				}
+
+				if tc.isAuth {
+					isAuthBool, err := getAuthenticatedImageStatus(ctx, client, tc.AuthImageStatus, *tc.pod)
+					if err != nil {
+						t.Fatal(err)
+					}
+					t.Logf("Is Credentials Provided:%v", isAuthBool)
```

Comment:
I think this code and logic needs clarifying. The `AuthImageStatus` could presumably be `failed` if we are testing that the wrong password/no password can't successfully pull the image, but the `getAuthenticatedImageStatus` doesn't seem to reflect that case. We also want to log, what is the expected status and what terminating reason we got.

Reply:
Since I wrote this it looks like the `getAuthenticatedImageStatus` got updated, so I'm not sure if the comment is exactly true, but I think at least the logging needs improvement